redbot/cogs/audio/audio.py

```diff
@@ -2586,6 +2611,24 @@ def _track_creator(player, position=None, other_track=None):
             track_obj[key] = value
         return track_obj
 
+    def _url_check(self, url):
+        valid_tld = [
+            "youtube.com",
+            "youtu.be",
+            "soundcloud.com",
+            "bandcamp.com",
+            "vimeo.com",
+            "mixer.com",
+            "twitch.com",
+        ]
+        site_check = []
+        for site in valid_tld:
+            if site in url.lower():
+                site_check.append(True)
```

Comment:
And on matching a solution that was, this can just be `return True` here, and everything else below it in the function would be omitable